examples/create_alert.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+
+  tasks:
+  - name: Create an alert in ManageIQ
+    manageiq_alert:
+      miq_url: http://miq.example.com
+      miq_username: admin
+      miq_password: secret
+      miq_verify_ssl: false
+      description: Test Alert 01
+      options:
+        notifications:
+          delay_next_evaluation: 600
+          evm_event: {}
+      entity: node
+      expression:
+        eval_method: dwh_generic
```

Comment:
> and IIUC it takes 2 manageiq_provider definitions for openshift+linked dwh, I'd throw those in as well...

@cben Not sure what you meant here, why 2?

> better have an example with some notifications if those are supported.

@moolitayer Do you have an example of alert with notidication?